rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRAStatusHolder.java

```diff
@@ -76,6 +78,10 @@ public LRAStatus getStatus() {
         return lraStatus;
     }
 
+    public String getLraStatusReason() {
+        return isRecovering() ? lraStatusReason : "N/A";
```

Comment:
I think it could be good to move this `N/A` constant under `LRAStatusReason`.
The change of the text of the reason may be done at one place while the constant is used on the other place as well.

Reply:
A question: why is exposed the reason only in case the LRA is recovering? I can't get it. Does it mean that the `LRAStatusReason` describes only some failure states?

Reply:
Status reasons are constrained to the belong to a fixed set so use a data type (ie don't use free format strings).

Also, why would the reason for being in a particular state ever *not* be applicable? Perhaps you meant "Unknown".

Reply:
I see. Good.